src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -773,31 +768,39 @@ func (c controller) waitForOLMOperators() bool {
 		return true
 	}
 	for _, operator := range operators {
-		csvName, err := c.kc.GetCSVFromSubscription(operator.Namespace, operator.SubscriptionName)
-		if err != nil {
-			c.log.WithError(err).Warnf("Failed to get subscription of operator %s", operator.Name)
-			continue
-		}
 
-		csv, err := c.kc.GetCSV(operator.Namespace, csvName)
-		if err != nil {
-			c.log.WithError(err).Warnf("Failed to get %s", operator.Name)
-			continue
-		}
+		//FIXME: We check the SC of the OCS operator WA bug 1968606
+		//Remove this code when bug 1968606 is fixed
+		sc, err := c.kc.GetStorageCluster()
```

Comment:
What if current `operator` is CNV, you chech StorageCLuster which is available, and update the CNV operator by the StorageCluster status. I think the previous option was correct.

Reply:
I have added the operator.namespace, so if its a cnv or any other operator, StorageCluster will return nil, It's a hack. The other option can be to wait after applying the post manifest install yaml, but we don't know how long we have to wait as it may depend on the cluster.